src/taipy/rest/api/resources/datanode.py

```diff
@@ -206,8 +246,24 @@ class DataNodeReader(Resource):
     get:
       tags:
         - api
-      summary: Read a data node content
-      description: Return a content of a data node
+      summary: Read a data node
+      description: |
+        Return the data read from a data node by datanode_id. If the data node does not exist, a 404 error is returned.
```

Comment:
'Return the data read from a data node by datanode_id.'
->
'The data is read from the data node whose identifier is \*datanode_id\*.'